extensions-utils/src/dependencies/mod.rs

```diff
@@ -27,18 +27,14 @@ pub fn execute_command(
         command.env("PATH", dfx_cache_path);
     }
     command.stdin(process::Stdio::null());
-    command.output().map_err(anyhow::Error::from).and_then(
-        |output| -> Result<String, anyhow::Error> {
-            if output.status.success() {
-                Ok(String::from_utf8_lossy(&output.stdout).into_owned())
-            } else {
-                Err(anyhow!(
-                    "Command failed:\n{:?}\nStdout:\n{}\n\nStderr:\n{}",
-                    command,
-                    String::from_utf8_lossy(&output.stdout),
-                    String::from_utf8_lossy(&output.stderr)
-                ))
-            }
-        },
-    )
+    command.stdout(process::Stdio::inherit());
+    command.stderr(process::Stdio::inherit());
+
+    let status = command.status().map_err(anyhow::Error::from)?;
```

Comment:
Is it intentional to skip the "Command execution failed: " command formatting if the `?` here exits the method?